apps/ledger-live-mobile/src/newArch/features/Accounts/components/AccountsListView/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback, useMemo } from "react";
+import { FlashList, FlashListProps } from "@shopify/flash-list";
+import useAccountsListViewModel, { type Props } from "./useAccountsListViewModel";
+import { Account } from "@ledgerhq/types-live";
+import { Flex } from "@ledgerhq/native-ui";
+import { Pressable } from "react-native";
+import AccountItem from "./components/AccountItem";
+import globalSyncRefreshControl from "~/components/globalSyncRefreshControl";
+
+type ViewProps = ReturnType<typeof useAccountsListViewModel>;
+
+const View: React.FC<ViewProps> = ({ accountsToDisplay, isSyncEnabled, onAccountPress }) => {
+  const List = useMemo(() => {
+    return isSyncEnabled ? globalSyncRefreshControl<FlashListProps<Account>>(FlashList) : FlashList;
+  }, [isSyncEnabled]);
+
+  const renderItem = useCallback(
+    ({ item }: { item: Account }) => (
+      <Pressable
+        style={({ pressed }: { pressed: boolean }) => [
+          { opacity: pressed ? 0.5 : 1.0, marginVertical: 12 },
+        ]}
+        hitSlop={6}
+        onPress={onAccountPress.bind(null, item)}
+      >
+        <Flex height={40} flexDirection="row" columnGap={12}>
+          <AccountItem account={item} balance={item.spendableBalance} />
+        </Flex>
+      </Pressable>
+    ),
+    [onAccountPress],
+  );
+
+  return (
+    <List
```

Comment:
how will you manage the maxDisplayed item when you will reuse the component in different area ?

Reply:
There is a props for it that slice the displayedAccounts

Reply:
That way you can pass a number (for our case 5) and the list will be sliced to have only 5 accounts/assets. I didn't add it inside the navigator params since we won't have the necessity to slice the number of accounts/assets to render on a screen

Reply:
sounds good for me